portal-web/docroot/html/portal/layout.jsp

```diff
@@ -16,8 +16,16 @@
 
 <%@ include file="/html/portal/init.jsp" %>
 
+<%
+Boolean is404Page = (Boolean)request.getAttribute("IS_404_PAGE");
```

Comment:
@AliciaGarciaGarcia @4lejandrito The parameter/variable name feels a bit off. What about using "layoutNotFound" or something similar?

Reply:
@adolfopa I am already reviewing this.

Reply:
The value is true if _layout.friendly.url.page.not.found_ is not empty, but the _layoutNotFound_ name is not valid...

The corner case that we are trying to is that
- The layout it's not found, the property _layout.friendly.url.page.not.found_ exists but the layout that is redirected is invalid (not exists) so on https://github.com/liferay-lima/liferay-portal/pull/1081/files#diff-8de9d78cf9232003a40dac7e2af521aa1ee3a6bca556bd9bb3224d9bdb6d8d5eR6698 we remove it in the next iteration.

if is a _layout.friendly.url.page.not.found_ > a 404 redirection, we do not show the status/error page, but any other idea, is welcomed